src/Microsoft.DotNet.ProjectJsonMigration/Rules/MigrateTFMRule.cs

```diff
@@ -75,20 +75,9 @@ private static string GetDisplayVersion(Version version)
             return sb.ToString();
         }
 
-        // TODO: When we have this inferred in the sdk targets, we won't need this
-        private AddPropertyTransform<NuGetFramework> OutputPathTransform =>
-            new AddPropertyTransform<NuGetFramework>("OutputPath",
-                f => $"bin/$(Configuration)/{f.GetShortFolderName()}",
-                f => true);
-
-        private AddPropertyTransform<NuGetFramework> FrameworkIdentifierTransform =>
-            new AddPropertyTransform<NuGetFramework>("TargetFrameworkIdentifier",
-                f => f.Framework,
-                f => true);
-
-        private AddPropertyTransform<NuGetFramework> FrameworkVersionTransform =>
-            new AddPropertyTransform<NuGetFramework>("TargetFrameworkVersion",
-                f => "v" + GetDisplayVersion(f.Version),
-                f => true);
+        private AddPropertyTransform<IEnumerable<NuGetFramework>> FrameworksTransform =>
+            new AddPropertyTransform<IEnumerable<NuGetFramework>>("TargetFrameworks",
+                frameworks => string.Join(";", frameworks.Select(f => f.GetShortFolderName())),
```

Comment:
No problem for netstandard, netcoreapp. In fact, my initial change will only work for those. There's other work in the targets to build for other things. So, we can deal with the others as they come online.

An example is .,NETPortable,Version=v4.6,Profile=Profile44 which is net451+win8 in short form. We can't handle that without the user providing both names.

I'll file an issue for the frameworks other than netcoreapp and netstandard and we can note the changes needed for PLIB there.